libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -541,6 +541,16 @@ bool AP_Mount::get_attitude_euler(uint8_t instance, float& roll_deg, float& pitc
     return true;
 }
 
+bool AP_Mount::get_target(uint8_t instance, float& roll_deg, float& pitch_deg, float& yaw_deg)
```

Comment:
Let's name this "get_attitude_target" to clarify that the target is an attitude because there are actually 3 supported target types (rate, angle and location).

Reply:
.. or maybe we should name it similar to the method to get the current angle.. which is "get_attitude_euler".  So mayget "get_target_attitude_euler" would be good.